configure.ac

```diff
@@ -662,6 +665,8 @@ AS_IF([test "x$with_slurm" != no],[
 ])
 AM_CONDITIONAL([HAVE_SLURM], [test "x$have_slurm" = xyes])
 
+OPTION_DEFAULT_DISABLE([spank_subscriber], [ENABLE_SPANK_SUBSCRIBER], [Builds sampler that subscribes to spank plugin])
+
```

Comment:
Why do we disable slurm/spank plugin by default? I think it is already automatically determined by the `with slurm` option above. This will NOT build slurm by default unless you explicitly enable it.

Reply:
we want the option to have slurm but not build the plugin.

Reply:
That is fine, but with this solution, you should have to explicitly disable it, otherwise the vast majority of users will have to explicitly enable it.